src/flat_list.jsx

```diff
@@ -1,16 +1,65 @@
 import React from "react";
```

Comment:
Same Points as mentioned in example file. 'example/src/app/component/flat_list.jsx'